stdlib/zipfile.pyi

```diff
@@ -274,6 +274,12 @@ if sys.version_info >= (3, 8):
             def joinpath(self, *other: StrPath) -> Path: ...
         else:
             def joinpath(self, add: StrPath) -> Path: ...  # undocumented
+        if sys.version_info >= (3, 12):
+            def glob(self, pattern: str) -> map[Self]: ...
+            def rglob(self, pattern: str) -> map[Self]: ...
+            def is_symlink(self) -> Literal[False]: ...
+            def relative_to(self, other: Self, *extra: StrPath) -> str: ...
+            def match(self, path_pattern: str) -> bool: ...
```

Comment:
There are also new `__eq__` and `__hash__` methods. `__eq__` methods are important for mypy's `--strict-equality` check; pyright uses `__hash__` methods to determine whether objects can be stored as dictionary keys or not.

```suggestion
            def match(self, path_pattern: str) -> bool: ...
            def __eq__(self, other: object) -> bool: ...
            def __hash__(self) -> int: ...
```